src/Umbraco.Web/Editors/PreviewController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Web.Mvc;
+using Umbraco.Web.Features;
+using Umbraco.Web.Mvc;
+
+namespace Umbraco.Web.Editors
+{
+    [UmbracoAuthorize]
+    [DisableBrowserCache]
+    public class PreviewController : Controller
+    {
+        private const string ViewsPath = "~/Umbraco/Views/Preview/";
```

Comment:
This *cannot* be hard coded, the /umbraco path can be changed, this needs to be done like `GlobalSettings.Path.EnsureEndsWith('/') + ...`